rapids-cmake/export/write_dependencies.cmake

```diff
@@ -74,6 +74,11 @@ function(rapids_export_write_dependencies type export_set file_path)
   # Do we need a Template header?
   set(_RAPIDS_EXPORT_CONTENTS)
   if(uses_cpm)
+    # Include download_with_retry.cmake in the exported dependencies first
+    file(READ "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../cmake/download_with_retry.cmake"
+         download_with_retry_logic)
+    string(APPEND _RAPIDS_EXPORT_CONTENTS ${download_with_retry_logic})
+
     file(READ "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../cpm/detail/download.cmake" cpm_logic)
```

Comment:
I'm confused, I don't see `cpm/detail/download.cmake` in here: https://github.com/rapidsai/rapids-cmake/tree/branch-25.06/rapids-cmake/cmake/detail

Is this a bug?

Reply:
Thank you. I read this about 5 times to check myself and couldn't see the difference. 🙃